doc/01-authoring-azure-cli-commands.md

```diff
@@ -32,11 +68,37 @@ Minimal file must exist in
       namespace: azure.mgmt.healthcareapis
     ```
 
+Minimal **readme.az.md** file must exist: 
+
+    ## AZ
+
+    These settings apply only when `--cli` is specified on the command line.
```

Comment:
here should be:  These settings apply only when `--az` is specified on the command line. right

Reply:
yup